src/Jackett.Common/Utils/Clients/WebByteResult.cs

```diff
@@ -2,6 +2,5 @@ namespace Jackett.Common.Utils.Clients
 {
     public class WebClientByteResult : BaseWebResult
     {
-        public byte[] ContentBytes { get; set; }
```

Comment:
This class is empty now. I suppose you that are going to combine the classes `WebByteResult`, `WebStringResult`, `BaseWebResult` and `WebClientResult` all in one don't you? IMHO that should be the way, unless there are technical problems. I approve this PR but I want to know.

Reply:
Yes, I will be combining them into a single WebResult class by the end. Problem is that there are references that need refactoring currently, and I didn't want to clog this PR up too much. Only the ones that ReSharper deemed safe to pull up have been changed to BaseWebResult for now.